### ISSUE TITLE
Error: Failed to download the requested certificate(s) for chain e476187f6ddfeb9d in order to advance to the next height 5 when create application

### ISSUE THREAD

**Issue by kikakkz** [linera-io/linera-protocol#882]
**Issue**
`Error: Failed to download the requested certificate(s) for chain e476187f6ddfeb9d in order to advance to the next height 5` when create application

**Steps**
1 `./scripts/run_local.sh`
2 `linera --wallet "$LINERA_WALLET" --storage "$LINERA_STORAGE" publish-bytecode ../res-peer/target/wasm32-unknown-unknown/release/res-peer_{contract,service}.wasm` here application could be anyone
3 `linera --wallet "$LINERA_WALLET" --storage "$LINERA_STORAGE" create-application e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000 --json-argument '{"total_supply":"10.","amount_alive_ms":70}'`
4 stop local net and do not stop service
5 run local net
6 stop service
7 `linera --wallet "$LINERA_WALLET" --storage "$LINERA_STORAGE" create-application e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000 --json-argument '{"total_supply":"10.","amount_alive_ms":70}'`

**Expectation**
We should be able to create application.

**Result**
```
kk@lenovo:~/linera-project/linera-protocol$ linera --wallet "$LINERA_WALLET" --storage "$LINERA_STORAGE" create-application e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000 --json-argument '{"total_supply":"10.","amount_alive_ms":70}'
2023-07-06T01:39:31.815349Z  WARN linera_service::storage: Using existing database "/home/kk/linera-project/linera-protocol/target/debug/linera.db"
2023-07-06T01:39:31.846981Z  INFO linera: Processing arguments...
2023-07-06T01:39:31.847062Z  INFO linera: Synchronizing...
Error: Failed to download the requested certificate(s) for chain e476187f6ddfeb9d in order to advance to the next height 5
```

**Comment by christos-h** [comment]
@kikakkz for now, restarting the local net completely nukes the existing network. The new local net starts with a new genesis configuration, new wallets and empty  validator states.

**Comment by afck** [comment]
I believe this is working as expected? It's not possible to use a wallet from one network (the first local net you started) with another network (the second one).

**Comment by kikakkz** [comment]
> I believe this is working as expected? It's not possible to use a wallet from one network (the first local net you started) with another network (the second one).

Yeah it's expected. It's quite early issue when I didn't understand the network clear, 😄
